WellCoffeeInventoryAPI/src/main/java/com/team_lightyear/WellCoffeeInventoryAPI/models/Item.java

```diff
@@ -62,20 +63,29 @@ public class Item {
     public Item() {
     }
 
+        //Constructor String
+    public Item(int id, String name, Integer quantity, Double price, String location, String description, String comment, Category category) {
+        this.id = id;
+    }
+
     // Constructor
     public Item(String name, Integer quantity, Integer minQuantity, Double price, String location,
-                String description, Category category, String amazonProductId, String image) {
+                String description, Category category, String amazonProductId, String image, String comment) {
+
         this.name = name;
         this.quantity = quantity;
         this.minQuantity = minQuantity;
         this.price = price;
         this.location = location;
         this.description = description;
+        this.comment = comment;
```

Comment:
Looks like there is an extra comment line!

Reply:
Good catch. New commit just pushed.